docs/tutorial/force_calibration/force_calibration.rst

```diff
@@ -0,0 +1,236 @@
+""""""""""""""""""""""""""""
+
+Proximity of the bead to the flowcell surface leads to an increase in the drag force on the bead.
+
+When performing experiments near the surface, it is recommended to provide the `distance_to_surface` argument.
+This should be the distance from the center of the bead to the surface of the flowcell in microns.
+Since it can be challenging to determine this distance, **it is recommended to use active calibration
+when calibrating near the surface,** since this makes calibration far less sensitive to mis-specification
+of the bead diameter, viscosity and height.
+
+Active or passive?
+""""""""""""""""""
+
+Passive calibration depends strongly on the values set for the bead diameter, distance to the surface and viscosity.
+When these are not well known, passive calibration can be inaccurate.
+During active calibration, the trap or nanostage is oscillated sinusoidally, leading to additional
+bead motion that can be detected and used to calibrate the displacement signal.
+Because of this extra information, active calibration does not rely as strongly on the calibration parameters.
+
+**Active calibration is highly recommended when the bead is close to the surface,**
+as it is less sensitive to the distance to the surface and bead diameter.
+**It is also recommended when the viscosity of the medium or the bead diameter are poorly known.**
+An example of a basic active calibration with a single bead can be found :ref:`here<active_calibration_tutorial>`.
+
+.. important::
+    One thing to be aware of is that active calibration with two beads is more complex and currently
+    requires :ref:`manual steps in Pylake<bead_bead_tutorial>`.
+
+Axial force
+"""""""""""
+
+When calibrating axial forces, it is important to set the `axial` flag to `True`.
+**Note that we currently do not support hydrodynamically correct models for axial calibrations.**
+Setting `axial` to `True` means you have to set `hydrodynamically_correct` to `False`.
+
+Technical considerations
+************************
+
+Sensor
+""""""
+
+In addition to the model that describes the bead's motion, it is important to take into account the
+:ref:`characteristics of the sensor<diode_theory>` used to measure the data.
+A silicon diode sensor is characterized by two parameters, a "relaxation factor" `alpha` and frequency `f_diode`.
+These parameters can either be estimated along with the other parameters or measured independently.
+
+When the diode frequency is fitted, care must be taken that the corner frequency of the power spectrum `fc` is
+:ref:`lower than the estimated diode frequency<high_corner_freq>`.
+You can check whether the frequency was estimated from the calibration
+data by checking the property :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.fitted_diode`.
+When this property is `True`, it means that the diode parameters were not fixed during the fit.
+This means that you should be careful when calibrating small beads at high laser powers.
```

Comment:
this sentence is very vague, I can't tell if there are particular recommendations in this case or if you should just avoid it?

Reply:
Good point. Rephrased to:
```
This means that you should try to avoid calibrating small beads at high laser powers.
```